pimd/pim_pim.c

```diff
@@ -331,8 +331,13 @@ static int pim_sock_read(struct thread *t)
 	struct interface *ifp, *orig_ifp;
 	struct pim_interface *pim_ifp;
 	int fd;
+#if PIM_IPV == 4
 	struct sockaddr_in from;
```

Comment:
Use `struct sockaddr_storage` instead for both IPv4 and IPv6 for simplicity. (`sockaddr_storage` is guaranteed to work for all AFs, IPv4/IPv6/UNIX/etc.)

Reply:
Ack